index.html

```diff
@@ -5,30 +5,67 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/index.css">
     <title>Document</title>
 </head>
 
+
 <body>
-    <h1>Roberto Gavino Flores</h1>
+    <header>
+        <h1>Roberto Gavino</h1>
+        <nav>
+            <ul>
+                <li>
+                    <a href="#about_me">About</a>
+                </li>
+                <li>
+                    <a href="#experience">Experience</a>
+                </li>
+                <li>
+                    <a href="#skills_section">Skills</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+
     <section>
-        <h1>About me</h1>
-        <p>Hello there my name is Roberto Gavino, I'm currently working as a construction worker but on my free time, I
-            like
-            to learn about coding.</p>
+        <h1 id="about_me">About me</h1>
+        <p>Formely brought to the USA at 10 years old, I chose work over school and found my stride in various
+            restaurant
+            roles. After a few years, I transitioned to construction, accumulating 4 years of experience. Currently,
+            I'm
+            delving into coding, fueled by the ambition to evolve professionaly. A devoted dad to 9-year-old
+            daughter
+            and a
+            2-year-old son, my journey weaves between construction site, coding projects, and the joys of
+            fatherhood.
+        </p>
     </section>
     <section>
-        <h1>Experience</h1>
-        <p>So far i have little to none Experience but I’m working on it.</p>
+        <h1 id="experience">Experience</h1>
+        <p>Though my professional roots lie in construction and restaurant role, I've been immersing myself in the
+            world
+            of
+            coding, particularly focusing on JavaScript and mastering the trio of HTML, CSS and JavaScript. Despite
+            limited
+            experience in coding compared to my extensive background in other fields, my enthusiasm for learning and
+            evolving in the coding realm is palpable.
+            Excited about the possibilities this new skill set brings, I egerly embrace each challenge that comes my
+            way.
+        </p>
     </section>
     <section>
-        <h1>Connect</h1>
+        <h1 id="connect">Connect</h1>
         <a href="https://github.com/dread-cmd">Github</a>
     </section>
-    <section id="skills">
-        <ul></ul>
+    <section>
+        <h1 id="skills_section">Skills</h1>
+        <div id="skills">
+            <ul id="skills_ul"></ul>
+        </div>
     </section>
     <section>
-        <h2>Leave me a message</h2>
+        <h2 id="message">Leave me a message</h2>
         <form>
```

Comment:
The form is missing labels.  It's important to load your page into a browser and see what it looks like.